examples/extensions/add_prices.py

```diff
@@ -0,0 +1,171 @@
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.v2.services.campaign_service_client import (
+    CampaignServiceClient
+)
+
+
+def main(client, customer_id, campaign_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID str.
+        campaign_id: a campaign ID str.
+    """
+    # Create the price extension feed item
+    price_feed_item = client.get_type('PriceFeedItem', version='v2')
+    price_feed_item.type = (
+        client.get_type('PriceExtensionTypeEnum', version='v2').SERVICES)
+    # Optional: set price qualifier
+    price_feed_item.price_qualifier = (
+        client.get_type('PriceExtensionPriceQualifierEnum').FROM)
+    price_feed_item.tracking_url_template.value = (
+        'http://tracker.example.com/?u={lpurl}')
+    price_feed_item.language_code.value = 'en'
+
+    # To create a price extension, at least three price offerings are needed.
+    price_feed_item.price_offerings.extend([
+        create_price_offer(client,
+                    'Scrubs',
+                    'Body Scrub, Salt Scrub',
+                    60000000,  # 60 USD
+                    'USD',
+                    client.get_type('PriceExtensionPriceUnitEnum').PER_HOUR,
+                    'http://www.example.com/scrubs',
+                    'http://m.example.com/scrubs'),
+        create_price_offer(client,
+                    'Hair Cuts',
+                    'Once a month',
+                    75000000,  # 75 USD
+                    'USD',
+                    client.get_type('PriceExtensionPriceUnitEnum').PER_MONTH,
+                    'http://www.example.com/haircuts',
+                    'http://m.example.com/haircuts'),
+        create_price_offer(client, 'Skin Care Package',
+                    'Four times a month',
+                    250000000,  # 250 USD
+                    'USD',
+                    client.get_type('PriceExtensionPriceUnitEnum').PER_MONTH,
+                    'http://www.example.com/skincarepackage')
+    ])
+
+    # Create a customer extension setting using the previously created
+    # extension feed item. This associates the price extension to your
+    # account.
+    extension_feed_item_operation = (
+        client.get_type('ExtensionFeedItemOperation', version='v2'))
+    extension_feed_item = extension_feed_item_operation.create
+    extension_feed_item.extension_type = (
+        client.get_type('ExtensionTypeEnum').PRICE)
+    extension_feed_item.price_feed_item.CopyFrom(price_feed_item)
+    extension_feed_item.targeted_campaign.value = (
+        CampaignServiceClient.campaign_path(customer_id, campaign_id))
+    extension_feed_item.ad_schedules.extend([
+        create_ad_schedule_info(client,
+            client.get_type('DayOfWeekEnum', version='v2').SUNDAY,
```

Comment:
Given that you use these several times, consider making a local variable for the enums and using that.

Reply:
fixed.